src/components/VideoHome/VideoHome.component.jsx

```diff
@@ -0,0 +1,907 @@
+import React from 'react';
+import styled from 'styled-components';
+import VideoItem from '../VideoItem';
+
+const Wrapper = styled.div`
+  display: inline-block;
+`;
+const VideoList = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+`;
+
+function VideoHome() {
+  const ytdata = {
+    kind: 'youtube#searchListResponse',
+    etag: 'LRviZfd_p3HDDD2uBk5Qv7zaEQU',
+    nextPageToken: 'CBkQAA',
+    regionCode: 'MX',
+    pageInfo: {
+      totalResults: 2323,
+      resultsPerPage: 25,
+    },
+    items: [
```

Comment:
This variable can be in its own file and just import it in this component, in a future challenge you will use the YouTube API instead of the mocked data